controllers/secret_controller.go

```diff
@@ -417,6 +445,216 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 	}
 }
 
+// createGoalertRoute creates an AlertManager Route for GoAlert in memory.
+func createGoalertRoute(namespaceList []string) *alertmanager.Route {
+
+	// order matters.
+	// these are sub-routes.  if any matches it will not continue processing.
+	// 1. route anything we consider critical to "GoAlert High"
+	// 2. route anything we want to silence to "null"
+	// 3. route anything that should be a warning to "GoAlert Low"
+	// 4. route anything we want to go to GoAlert
+	//
+	// the Route docs can be read at https://prometheus.io/docs/alerting/latest/configuration/#matcher
+	goalertSubroutes := []*alertmanager.Route{
```

Comment:
/lgtm cancel
/hold

Is there a way we can dedupe this? Having two copies of this alerting list for goalert vs pagerduty is going to cause issues.

Reply:
I'm curious how this will cause issues?  There is a chance we will need a different configuration in FedRAMP than commercial.